src/v1/controllers/VerifyWebauthnController.js

```diff
@@ -138,7 +138,8 @@ export default FormController.extend({
               self.webauthnAbortController = null;
             });
         });
-      });
+      })
+        .catch(() => {});
```

Comment:
I don't have a ton of context here so ignore if it's not relevant: should we log any exceptions here before swallowing them?

Reply:
Good question. We discussed that in office hours. The exceptions are handled elsewhere and displayed to the user as error messages, so these unhandled rejections are only polluting the console. You can see that the previously failing specs each contain assertions that an error message is displayed.